homeassistant/components/homeassistant/translations/en.json

```
@@ -1,3 +1,17 @@
 {
-    "title": "Home Assistant"
+    "system_health": {
+        "info": {
+            "arch": "CPU Architecture",
+            "dev": "Development",
+            "docker": "Docker",
+            "hassio": "HassOS",
+            "installation_type": "Installation Type",
+            "os_name": "Operating System Name",
```

```suggestion
            "os_name": "Operating System Family",
```

In most cases, this will just show "Linux"